modules/combined/tests/frictional_contact/sliding_elastic_blocks_2d/sliding_elastic_blocks_2d.i

```diff
@@ -177,19 +211,19 @@
 
 
 
-  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart'
-  petsc_options_value = 'hypre    boomeramg      101'
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  petsc_options_value = 'lu     superlu_dist'
```

Comment:
Please add "superlu = true" in the tests file for this test (and any others that you switch to using `-pc_factor_mat_solver_package superlu_dist`.